src/main/java/com/minecolonies/coremod/entity/pathfinding/Pathfinding.java

```diff
@@ -91,153 +90,156 @@ public static void debugDraw(final double frame)
-    private static void renderDebugText(@NotNull final Node n, final float f1)
+    private static void renderDebugText(@NotNull final Node n)
     {
         final String s1 = String.format("F: %.3f [%d]", n.getCost(), n.getCounterAdded());
         final String s2 = String.format("G: %.3f [%d]", n.getScore(), n.getCounterVisited());
         final FontRenderer fontrenderer = Minecraft.getMinecraft().fontRendererObj;
-        GL11.glPushAttrib(GL11.GL_ALL_ATTRIB_BITS);
-        GL11.glPushMatrix();
-        GL11.glTranslatef(0.0F, 0.75F, 0.0F);
-        GL11.glNormal3f(0.0F, 1.0F, 0.0F);
+        GlStateManager.pushAttrib();
+        GlStateManager.pushMatrix();
+        GlStateManager.translate(0.0F, 0.75F, 0.0F);
+        GlStateManager.glNormal3f(0.0F, 1.0F, 0.0F);
 
         final RenderManager renderManager = Minecraft.getMinecraft().getRenderManager();
-        GL11.glRotatef(-renderManager.playerViewY, 0.0F, 1.0F, 0.0F);
-        GL11.glRotatef(renderManager.playerViewX, 1.0F, 0.0F, 0.0F);
-        GL11.glScalef(-f1, -f1, f1);
-        GL11.glTranslatef(0.0F, (float) (0.25D / f1), 0.0F);
-        GL11.glDepthMask(false);
-
-        final Tessellator tessellator = Tessellator.getInstance();
-        final VertexBuffer VertexBuffer = tessellator.getBuffer();
-        GL11.glDisable(GL11.GL_TEXTURE_2D);
+        GlStateManager.rotate(-renderManager.playerViewY, 0.0F, 1.0F, 0.0F);
+        GlStateManager.rotate(renderManager.playerViewX, 1.0F, 0.0F, 0.0F);
+        GlStateManager.scale(-0.014F, -0.014F, 0.014F);
+        GlStateManager.translate(0.0F, 18F, 0.0F);
+
+        GlStateManager.depthMask(false);
+        GlStateManager.disableDepth();
+
+        GlStateManager.enableBlend();
+        GlStateManager.tryBlendFuncSeparate(
+                GlStateManager.SourceFactor.SRC_ALPHA,
+                GlStateManager.DestFactor.ONE_MINUS_SRC_ALPHA,
+                GlStateManager.SourceFactor.ONE,
+                GlStateManager.DestFactor.ZERO);
+        GlStateManager.disableTexture2D();
 
-        VertexBuffer.begin(GL11.GL_QUADS, DefaultVertexFormats.POSITION_TEX);
         final int i = Math.max(fontrenderer.getStringWidth(s1), fontrenderer.getStringWidth(s2)) / 2;
 
-        //that should set the colors correctly
-        VertexBuffer.pos((double) (-i - 1), -5.0D, 0.0D).color(0.0F, 0.0F, 0.0F, 0.25F).endVertex();
-        VertexBuffer.pos((double) (-i - 1), 12.0D, 0.0D).color(0.0F, 0.0F, 0.0F, 0.25F).endVertex();
-        VertexBuffer.pos((double) (i + 1), 12.0D, 0.0D).color(0.0F, 0.0F, 0.0F, 0.25F).endVertex();
-        VertexBuffer.pos((double) (i + 1), -5.0D, 0.0D).color(0.0F, 0.0F, 0.0F, 0.25F).endVertex();
+        final Tessellator tessellator = Tessellator.getInstance();
+        final VertexBuffer vertexBuffer = tessellator.getBuffer();
+        vertexBuffer.begin(GL11.GL_QUADS, DefaultVertexFormats.POSITION_COLOR);
+        vertexBuffer.pos((double) (-i - 1), -5.0D, 0.0D).color(0.0F, 0.0F, 0.0F, 0.25F).endVertex();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 5.0D to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.25F to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)